homeassistant/components/notify/pushover.py

```diff
@@ -51,7 +51,21 @@ def send_message(self, message="", **kwargs):
         """Send a message to a user."""
         from pushover import RequestError
 
+        data = kwargs.get(ATTR_DATA)
```

Comment:
Please don't modify the original data dictionary that is being passed in.

You can supply a second argument to `get` that will be returned if the key doesn't exist, that way you don't have to check for `None` on the next line.

```python
# Make a copy and use empty dict as default value
data = dict(kwargs.get(ATTR_DATA, {}))
```